DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Processors/Factories/DicomProcessorFactory.cs

```diff
@@ -22,8 +26,23 @@ public IAnonymizerProcessor CreateProcessor(string method, JObject settingObject
                 "remove" => new RemoveProcessor(),
                 "refreshuid" => new RefreshUIDProcessor(),
                 "keep" => new KeepProcessor(),
-                _ => null,
+                _ => CreateCustomProcessor(method)
```

Comment:
when shall we use setting?

Reply:
Some anonymize methods needs settings when creating. (perturb, substitute, dateShift, encrypt, cryptoHash and redact)